CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### ✨ Enhancements
 
+- Support `x in set()` and `x not in set()` preconditions by special-casing `set()` calls in the Z3 parser.
```

Comment:
Please modify this entry to reflect the increased scope of this change: you've added support for `set()`, `list()`, and `tuple()` function calls in preconditions. And this isn't a special case any more 😃